cronos-play/getting-started_unreal/quick-start/actors-and-blueprint-classes.md

```diff
@@ -0,0 +1,33 @@
+# Actors and Blueprint Classes
+
+## DefiWalletCoreActor
+
+**DefiWalletCoreActor** is a C++ Class that has all the features for interacting with Crypto.org Chain, Cronos Chain, and other EVM-compatible chains natively. Artists, designers, or programmers can utilise the powerful Blueprint Visual Scripting system in Unreal Engine to create DApps/DGames easily.&#x20;
```

Comment:
We don't have to mention other EVM-compatible chains, just 
Crypto.org Chain and Cronos Chain,

Reply:
Done